### ISSUE TITLE
'CLIP' object has no attribute 'save'

### ISSUE THREAD

**Issue by vgrabovets** [UKPLab/sentence-transformers#839]
Hi, thank you for this great repository.

When saving CLIPmodel there is an error: `'CLIP' object has no attribute 'save'`
sentence-transformers==1.0.3
torch==1.8.0
torchvision==0.9.0

```
from sentence_transformers import SentenceTransformer
model = SentenceTransformer('clip-ViT-B-32')
model.save('model')
```
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-14469469c6ff> in <module>
----> 1 model.save('assets/model')

/mnt/ubuntu_storage/projects/env/lib/python3.7/site-packages/sentence_transformers/SentenceTransformer.py in save(self, path)
    376             model_path = os.path.join(path, str(idx)+"_"+type(module).__name__)
    377             os.makedirs(model_path, exist_ok=True)
--> 378             module.save(model_path)
    379             contained_modules.append({'idx': idx, 'name': name, 'path': os.path.basename(model_path), 'type': type(module).__module__})
    380 

/mnt/ubuntu_storage/projects/env/lib/python3.7/site-packages/sentence_transformers/models/CLIPModel.py in save(self, output_path)
     74             json.dump({'model_name': model_name, 'tokenizer_name': tokenizer_name}, fOut, indent=2)
     75 
---> 76         self.model.save(os.path.join(output_path, model_name))
     77 
     78     @staticmethod

/mnt/ubuntu_storage/projects/env/lib/python3.7/site-packages/torch/nn/modules/module.py in __getattr__(self, name)
    946                 return modules[name]
    947         raise AttributeError("'{}' object has no attribute '{}'".format(
--> 948             type(self).__name__, name))
    949 
    950     def __setattr__(self, name: str, value: Union[Tensor, 'Module']) -> None:

AttributeError: 'CLIP' object has no attribute 'save'
```

**Comment by nreimers** [comment]
I missed to implement this method for the CLIP model.

It will be added in the next release.

**Comment by nreimers** [comment]
The issue is fixed and published in release 1.0.4
